cmd.sh

```diff
@@ -1,7 +1,10 @@
 #!/bin/bash
 
 if [ "$APP_ENV" == "DEVELOPMENT" ] || [ -z "$APP_ENV" ]; then
-    gunicorn -w 4 -b 0.0.0.0:8000 wsgi:app --reload --worker-class gevent
+    python health_audit_runner.py &
+    HEALTH_AUDIT_PID=$!
+    gunicorn -w 4 -b 0.0.0.0:8002 wsgi:app --reload --worker-class gevent
+    trap "kill -9 $HEALTH_AUDIT_PID" EXIT
```

Comment:
So you're basically running the health_audit_runner.py as a background process, trapping the PID for the utility and then killing it whenever the gunicorn instance dies. This may not always work as reliably as we hope, but it's fine because it's only for development. Fortunately, the container gets destroyed each time and this wouldn't really have much of an impact in production.

I do wonder a little bit though how this would scale. Meaning that you'd have two instances of the logger API running on the cluster, each with its own startup sequence and its own copy of the metrics file. There may be a possibility that someone would get different data depending on which node the load balancer hits. Not a big deal IMHO, but just something to think about and test./

Reply:
Right, I suppose you could have two Facet users (in the same data trust trust) looking at slightly different metrics, most likely, in the `last_accessed` timestamp. Of course, we might replace that field with different data, i.e., the last accessed timestamp of a client _who is not the logger api_.

For the `cmd.sh` script, I basically used the pattern you established in the data-resources-api: https://github.com/brighthive/data-resource-api/blob/master/cmd.sh

But, huh, it looks like we do not have a solution for running this script in a SANDBOX or PRODUCTION environment. How is this going to work in production?

Reply:
I updated #3 with info about this.